render_config.sh

```diff
@@ -4,7 +4,8 @@ VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
 TARGET_ENV=${2:-dev}
 
 FIRECLOUD_ACCOUNT_VAULT_PATH=secret/dsde/firecloud/${TARGET_ENV}/common/firecloud-account.json
-SERVICE_ACCOUNT_OUTPUT_FILE_PATH="$(dirname $0)"/src/test/resources/rendered/wsm-firecloud-account.json
+SERVICE_ACCOUNT_OUTPUT_FILE_PATH="$(dirname $0)"/src/main/resources/generated/wsm-firecloud-account.json
+SERVICE_ACCOUNT_TEST_OUTPUT_FILE_PATH="$(dirname $0)"/src/test/resources/rendered/wsm-firecloud-account.json
```

Comment:
Created https://broadworkbench.atlassian.net/browse/AS-455 to handle figuring out how SAs should work / where they should be rendered for WSM.

Reply:
I am leaving the test SA in for now, AS-455 will handle figuring out SAs